client/my-sites/importer/site-importer/site-importer-input-pane.jsx

```diff
@@ -374,6 +385,50 @@ class SiteImporterInputPane extends React.Component {
 		} );
 	};
 
+	renderUrlHint = () => {
+		switch ( this.props.importerData.engine ) {
+			case 'wix':
+				return (
+					<div>
+						<p>
+							{ this.props.translate( 'Please use one of following formats for the site URL:' ) }
+						</p>
+						<ul>
+							<li>
+								<span className="site-importer__site-importer-example-domain">example.com</span>
+								{ ' - ' }
+								{ this.props.translate( 'a paid custom domain' ) }
+							</li>
+							<li>
+								<span className="site-importer__site-importer-example-domain">
+									example-account.wixsite.com/my-site
+								</span>
+								{ ' - ' }
+								{ this.props.translate( 'a free domain that comes with every site' ) }
+							</li>
+						</ul>
+					</div>
+				);
+			case 'engine6':
+				return (
```

Comment:
It seems like there's a lot of shared/repeated shape to the two cases. At this point I think it's ok not to abstract but that might be something we consider doing in the future.